canopy/app/README.md

```diff
@@ -0,0 +1,43 @@
+<!-- Copyright 2019 Cargill Incorporated
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. -->
+
+# Canopy
+
+Canopy is a web application that hosts pluggable applications and tools built
+on Splinter.
+
+## Available Scripts
+
+In the project directory, you can run:
```

Comment:
I thought `project directory` meant `/canopy`.  But it is actually `/canopy/app`. So it could be more explicit about what the directory is here.